### PR TITLE
[update] Updated Google Colab links (#96) and notebook description

### LINKED ISSUE #96: google colab links broken
the google colab links link to the wrong location

### PR DESCRIPTION
**Pull request by ayushanand18**
## Overview
This PR aims to resolve #96 by updating Google Colab Links for the notebooks. It also updates the notebook description for biodiversity mapping notebook.

Thanks!

### CODE REVIEW

**Comment by ayushanand18** [comment]
**Update:** Some tests were failing due to the occurrence `id` that is being used in the tests. Fixed it with a small patch.

File: notebooks/biodiversity_mapping.ipynb [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "id": "view-in-github"
       },
       "source": [
-        "<a href=\"https://colab.research.google.com/github/ayushanand18/pyobis/blob/bio-mapping/notebooks/biodiversity_mapping.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+        "<a href=\"https://colab.research.google.com/github/iobis/pyobis/blob/master/notebooks/biodiversity_mapping.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
```
Comment (ocefpaf) [inline notebooks/biodiversity_mapping.ipynb:10]: I think that both @7yl4r and I have the power to rename the master branch to main now. Maybe let's take the opportunity do to this now and change these links only once?

Comment (7yl4r) [reply notebooks/biodiversity_mapping.ipynb:10]: good call! I just renamed the default branch from `master` to `main`. Please reconfigure your remote repos accordingly!

Comment (ayushanand18) [reply notebooks/biodiversity_mapping.ipynb]: Update. I have updated the badges `master`->`main`.

**Pull request merged**